nextcord/abc.py

```diff
@@ -117,8 +117,17 @@ class User(Snowflake, Protocol):
     ----------
     name: :class:`str`
         The user's username.
+    global_name: :class:`str`
+        The user's global name.
```

Comment:
Maybe add a reference to how this is called "Display Name" in the UI?

Reply:
We don't exactly add the references (like we haven't said that discriminator is 'tag' on the UI) so I'm not too sure about doing so

Reply:
Because discord themselves didn't know what to call it. Mentioned as <x> in the UI is mentioned in quite a few places.